metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -61,44 +60,59 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(
+  def fetchSettings(
       dependency: Dependency,
       scalaVersion: String
-  ): Settings =
-    new coursiersmall.Settings()
-      .withTtl(Some(Duration.Inf))
-      .withDependencies(List(dependency))
+  ): Fetch = {
+    val fetch = Fetch
+      .create()
+      .withDependencies(List(dependency): _*)
+      .addRepositories(
+        Repository.defaults().asScala.toList: _*
+      )
       .addRepositories(
         List(
-          coursiersmall.Repository.SonatypeReleases,
-          coursiersmall.Repository.SonatypeSnapshots
-        )
+          Repository.central(),
+          Repository.ivy2Local()
+        ): _*
```

Comment:
can we specify the parameters directly, without wrapping them in a list first?
like: `addRepositories(Repository.central, Repository.ivy2Local)`